### ISSUE TITLE
Where do I Todd-prod?

### ISSUE THREAD

**Issue by Toddsqui** [Toddsqui/todd-prod#1]
@ionicaBizau I created this new repository with the new todd-prod function in it. Do I NPM install from here? Let me know! Thanks!

**Comment by Toddsqui** [comment]
:confused:

**Comment by IonicaBizau** [comment]
@Toddsqui Do `npm init` to init the `package.json` file and then `npm i --save todd-sum` to add the `todd-sum` as dependency. :smile:

**Comment by Toddsqui** [comment]
I added an example. js file, but didn't end up using the sum module. It works right now, but I'm wondering how I can view this stuff on npm? Is there any connection between github and NPM that i am missing? Also, how would I add the sum module if I wanted to use it? :rotating_light:

**Comment by Toddsqui** [comment]
:de:  My mind isn't making the connection between NPM and Github.

**Comment by IonicaBizau** [comment]
> My mind isn't making the connection between NPM and Github.

There is no mandatory connection. You just `npm publish` the folder, basically.

> Also, how would I add the sum module if I wanted to use it?

`npm i --save todd-sum` in the todd-prod folder and then `require("todd-sum")` keeping in mind it returns a function that receives two arguments and it returns their sum.

**Comment by Toddsqui** [comment]
`https://www.npmjs.com/package/todd-prod`

Hmm, why isn't it showing up though? O_O

:sob:

**Comment by IonicaBizau** [comment]
@Toddsqui Did you `npm publish`?

**Comment by Toddsqui** [comment]
@IonicaBizau I thought that's what I did? That was your instruction when you told me to npm init. So that's what I did. :grey_question: 

What do you think?

**Comment by IonicaBizau** [comment]
@Toddsqui Just try it again: `npm publish` in the `todd-prod` directory.

**Comment by Toddsqui** [comment]
@IonicaBizau Okay! :clapper:

**Comment by Toddsqui** [comment]
@IonicaBizau 
": ERROR: No README data found!"

:crying_cat_face:

**Comment by IonicaBizau** [comment]
Hmm, did you run the command in the root of the project (in the same directory where the `README.md` lives)?

**Comment by Toddsqui** [comment]
Yah! Totally did! :rabbit:

**Comment by IonicaBizau** [comment]
Hmm, try reininting the `package.json` using `npm init` and then see if there is any change.

**Comment by Toddsqui** [comment]
This is what I get.

```
npm ERR! publish Failed PUT 403
npm ERR! Darwin 13.4.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "publish"
npm ERR! node v5.1.0
npm ERR! npm  v3.3.12
npm ERR! code E403

npm ERR! "You cannot publish over the previously published version 1.0.0." : todd-prod
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/macuser/github/todd-prod/npm-debug.log
```

**Comment by IonicaBizau** [comment]
Yes, because you already published 1.0.0: https://www.npmjs.com/package/todd-prod :joy:

**Comment by Toddsqui** [comment]
*dies* 
:laughing:

**Comment by IonicaBizau** [comment]
Anyways, the next step is to use the `todd-sum` as dependency in this package. Have fun. :crystal_ball:

**Comment by Toddsqui** [comment]
you mean, like... actually use the module, right? :skull:

**Comment by Toddsqui** [comment]
:sweat_smile:

**Comment by IonicaBizau** [comment]
Exactly. :smile:

**Comment by Toddsqui** [comment]
You can't call a module like a function. ex. total(); 

:anger:

**Comment by IonicaBizau** [comment]
You can, after `require`ing it. :stuck_out_tongue:

**Comment by Toddsqui** [comment]
Lies, all lies. 

:stuck_out_tongue_winking_eye:
